components/compositing/Cargo.toml

```diff
@@ -53,5 +53,5 @@ git = "https://github.com/servo/rust-core-text"
 git = "https://github.com/servo/gleam"
 
 [dependencies]
-url = "*"
-time = "*"
\ No newline at end of file
+url = "0.2.16"
```

Comment:
Hum. I couldn't figure out how to force Cargo to use a particular release of a crates.io package, however; it appeared to ignore the --precise argument.